src/server/rdb_load.cc

```diff
@@ -1600,7 +1603,8 @@ struct RdbLoader::ObjSettings {
   ObjSettings() = default;
 };
 
-RdbLoader::RdbLoader(ScriptMgr* script_mgr) : script_mgr_(script_mgr) {
+RdbLoader::RdbLoader(Service* service)
+    : service_{service}, script_mgr_{service == nullptr ? nullptr : service->script_mgr()} {
```

Comment:
I don't like the fact we're implicitly taking the script manager from the service, but maybe its better than passing both?